sys-libs/libcap-ng/libcap-ng-0.8.4-r1.ebuild

```diff
@@ -27,17 +27,19 @@ BDEPEND="python? ( >=dev-lang/swig-2 )"
 
 PATCHES=(
 	"${FILESDIR}"/${P}-swig.patch
+	# https://bugs.gentoo.org/928450
+	"${FILESDIR}"/${P}-slibtool.patch
 )
 
 src_prepare() {
 	default
 
 	if use prefix ; then
-		sed -i "s@cat /usr@cat ${EPREFIX}/usr@" bindings/python*/Makefile.am || die
 		# bug #668722
-		eautomake
+		sed -i "s@cat /usr@cat ${EPREFIX}/usr@" bindings/python*/Makefile.am || die
```

Comment:
We should drop this sed because it no longer does anything. See https://github.com/stevegrubb/libcap-ng/commit/a22d36a8c23dd40b39073ee0d3ca59f8eda160fe.

Reply:
I suppose we should pass this to configure instead:
```
--with-capability_header="${ESYSROOT}"/usr/include/linux/capability.h
```

Reply:
I added a second commit that adds `--with-capability_header` and removes the sed, I think this is what you meant?

Reply:
Yep, that's perfect. Thanks.